docs/content/en/usage/useDirectusCollections.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "useDirectusCollections"
+description: "Learn how to use Nuxt & Directus"
+position: 14
+category: "Usage"
+---
+
+> Check out the Directus [Collections](https://docs.directus.io/reference/collections/) documentation.
+
+### `listCollections`
```

Comment:
I would prefer if we renamed `listCollections` to `getCollections`.

Reply:
Renamed as requested - I was just naming it similar to the Directus name: https://docs.directus.io/reference/system/collections/#list-collections